build/sharpmake/src/base.cs

```diff
@@ -254,6 +254,11 @@ public override void Configure(RexConfiguration conf, RexTarget target)
     }
```

Comment:
Isn't this file becoming a bit bloated with all the different projects inside of it?

Reply:
maybe, but that shouldn't be part of this change.
no extra projects got added in this change